_posts/2023-06-06-Client-cert-auth-without-roles.adoc

```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+title: 'Using Certificate Based Authentication with Mutual TLS without Roles'
+date: 2023-06-06
+tags: cert-Generation server-Configuration deploying-App
+synopsis: How to deploy a simple web app on eildfly that uses mutual TLS connection and CLIENT_CERT to authenticate the client. 
+author: prarthonapaul
+---
+
+:toc: macro
+:toc-title:
+
+Mutual-TLS with client certificates is essential 
+for secure and authenticated communication between 
+clients and servers in various applications and systems. 
+It involved a handshake process during which both the client 
+and the server must present their certificate to each other 
+in order to establish a secure connection. This ensures that 
+both parties are authenticated before establushing the connection.  
+
+This example will walk you through how to secure a web application deployed to WildFly with mutual TLS and the CLIENT_CERT HTTP authentication mechanism. We will be doing so using CLIENT_CERT 
+HTTP authentication.In this example we will be using certificates 
```

Comment:
"We will be doing so using CLIENT_CERT HTTP authentication." can probably be removed since it's mentioned in the first sentence.